fixed sth but not sure if correct (#2332)
arbor-sim/arbor: doc/fileformat/cable_cell.rst

      (iexpr-def "my_iexpr" (radius 0.5))

   This expression identifies the radius iexpr with a scaling factor of 0.5.

> For any number of locset or region, an iexpr definitions can be grouped in a label dictionary as follows:

Reviewer: ```suggestion
Any  definition of a locset, region, or iexpr can be stored in a label dictionary as follows.
```
Label dictionaries give names to expressions that might be sensible to reuse. E.g.
```
ld = A.label_dict()
ld['term-dend'] = '(restrict (tag 3) (terminal))'
```
and we can then use these in placing things
```
decor.place('"term-dend"', A.synapse(...))
```
or in other expressions
```
ld['int'] = '(proximal-interval (locset "term-dend") 10)'
A.paint('"int", ...)
```
If any of that is not sufficiently explained here or in the labels discussion, we/I can update those places.
